### PR TITLE
add hasParameters to commands and remove title

### PR DESCRIPTION
**Pull request by DarkPark**

### CODE REVIEW

File: src/Command.ts [hunk 1]
```diff
@@ -29,8 +29,12 @@ abstract class Command {
 
     static examples: TCommandExampleList;
 
+    static hasParameters = false;
+
+
```
Comment (DeadByElpy) [inline src/Command.ts:34]:
```suggestion
```

File: src/commands/downlink/CorrectTime2000.ts [hunk 2]
```diff
@@ -54,11 +47,13 @@ class CorrectTime2000 extends Command {
...
     }
 
+
     static readonly id = COMMAND_ID;
 
     static readonly directionType = DOWNLINK;
 
-    static readonly title = COMMAND_TITLE;
+    static readonly hasParameters = true;
+
```
Comment (DeadByElpy) [inline src/commands/downlink/CorrectTime2000.ts:56]:
```suggestion
```

Comment (DeadByElpy) [reply src/commands/downlink/CorrectTime2000.ts:56]: in other files too, it's special separation or mistake?

Comment (DarkPark) [reply src/commands/downlink/CorrectTime2000.ts:56]: special separation

**Pull request merged**